pytket/phir/qtm_machine.py

```diff
@@ -21,20 +23,18 @@ class QtmMachine(Enum):
 QTM_MACHINES_MAP = {
     QtmMachine.H1_1: Machine(
         size=20,
+        gateset={OpType.Rz, OpType.PhasedX, OpType.ZZPhase},
         tq_options={0, 2, 4, 6, 8, 10, 12, 14, 16, 18},
         # need to get better timing values for below
         # but will have to look them up in hqcompiler
-        tq_time=3.0,
-        sq_time=1.0,
-        qb_swap_time=2.0,
+        timings=MachineTimings(tq_time=3.0, sq_time=1.0, qb_swap_time=2.0),
     ),
     QtmMachine.H1_2: Machine(
         size=12,
+        gateset={OpType.Rz, OpType.PhasedX, OpType.ZZPhase},
```

Comment:
Since this is a common gateset, perhaps this should also be exposed as a constant from this module or the `QtmMachine` class.